rllib/examples/random_parametric_agent.py

```diff
@@ -64,29 +66,31 @@ def set_weights(self, weights):
         pass
 
 
-# Create a new Trainer using the Policy and config defined above and a new
-# execution plan.
+# Backward compatibility, just in case users want to use the erroneous old name.
+RandomParametriclPolicy = RandomParametricPolicy
```

Comment:
is this really necessary?? :) :)
this is an example script ... folks probably shouldn't import an example script and use it as a library?